guides/common/modules/proc_customizing-registration-by-using-snippets.adoc

```diff
@@ -0,0 +1,23 @@
+[id="customizing-registration-by-using-snippets"]
+= Customizing registration by using snippets
+
+You can customize the registration process without editing the *Global Registration* template by creating snippets that the template includes if they exist.
+To add custom steps to registration, create one or both of the following snippets:
+
+* `before_registration` {endash} This snippet will be loaded and executed by the *Global Registration* template at the beginning of registration.
+* `after_registration` {endash} This snippet will be loaded and executed by the *Global Registration* template at the end of registration.
+
+You have to name the snippets precisely.
+Otherwise, they cannot be recognized.
```

Comment:
```suggestion
Otherwise, {Project} will not add them to the `Global Registration` template.
```
I highly recommend to add these details to specify what's going on.